docs/changelog.md

```diff
@@ -17,6 +17,8 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 -   Improved how spacing between the buttons was done
```

Comment:
Nice detailed change log.
On this project the convention seems to be simply have the name, and issue or PR link.
Perhaps move the details to the PR description.

Reply:
I moved this now into the PR.
(Sry, that it took me a while, but I had to write some uni exams this week)